DXRFixes/DeusEx/Classes/ElevatorMover.uc

```diff
@@ -23,17 +66,23 @@ function SetSeq(int seqnum)
     local int prevKeyNum;
     local float dist;
 
+    Initialize();
+
     if( MoveTime/2 < Level.TimeSeconds-lastTime )
```

Comment:
`MoveTime/2` means it's a pretty long wait when trying to change your mind about where to go

https://github.com/user-attachments/assets/5934952e-c6c5-4215-a7f5-d9313aefbe39

```suggestion
    if( MoveTime/10 < Level.TimeSeconds-lastTime )
```

Reply:
I think with that suggested change (or something in-between like if `MoveTime/8` feels better or something) and some more testing, we can probably accept this, it is an improvement

Reply:
I don't understand what that bit of code is doing.

Reply:
prevents retrigger from double clicking the button